molgenis/molgenis-emx2
# fix(catalogue): variable list should show +n when repeat info is set

closes #4259

What are the main changes you did:
- changes to variables page

how to test:
- browse to catalogue-demo -> TestNetwork -> Variables -> testRepeatVar_ and see that a +x is shown
- go to harmonisation matrix, click testRepeatVar_ and see "Repeated for'

todo:
- [x] fix the +1 bubble
- [x] fix the popup to also show repeatvar info
